src/main/java/org/checkerframework/languageserver/CFTextDocumentService.java

```diff
@@ -34,14 +34,17 @@
 
 /** This class does all the dirty works on source files. */
 public class CFTextDocumentService implements TextDocumentService, Publisher {
-
+    /** The logger for issuing information in the Checker Framework document service. */
     private static final Logger logger = Logger.getLogger(CFTextDocumentService.class.getName());
 
-    // Pattern of the range in CF message "lsp.type.information"
+    /** The pattern of the range in CF message "lsp.type.information". */
```

Comment:
Can you add references to https://github.com/eisop/checker-framework/blob/3ed0c114c3d686eadc803207640487e86d1d086e/framework/src/main/java/org/checkerframework/framework/source/messages.properties#L3 and https://github.com/eisop/checker-framework/blob/3ed0c114c3d686eadc803207640487e86d1d086e/framework/src/main/java/org/checkerframework/framework/util/TypeInformationPresenter.java#L138 to make it easier to find this again.

Reply:
Already add the reference.